quickstarts/maven/spring-boot-helm/pom.xml

```diff
@@ -51,17 +63,66 @@
       <groupId>org.springframework.boot</groupId>
       <artifactId>spring-boot-starter-actuator</artifactId>
     </dependency>
-
   </dependencies>
 
   <build>
     <plugins>
-
       <plugin>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-maven-plugin</artifactId>
       </plugin>
-
+      <plugin>
+        <groupId>org.eclipse.jkube</groupId>
+        <artifactId>kubernetes-maven-plugin</artifactId>
+        <version>${jkube.version}</version>
+        <configuration>
+          <resources>
+            <labels>
+              <all>
+                <testProject>spring-boot-with-yaml-label-for-all</testProject>
+              </all>
+            </labels>
+          </resources>
+          <helm>
+<!--            <chart>This is a Test</chart>-->
```

Comment:
is this leftover from deleted profiles?

Reply:
No, it was commented-out because in the readme it points out how to change the charts using XML config.

We'll probably need to create further examples because now there are more features than before and it is hard to concentrate everything in a single quick start project